plugins/modules/openshift_versions.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: openshift_versions
+
+short_description: Query supported OpenShift versions
+
+version_added: "1.0.0"
+
+description: Retrieves the list of supported OpenShift versions.
+
+
+options:
+  version:
+    description: Version of OpenShift
+    required: False
+    type: str
+
+  only_latest:
+    description: Retrieve only latest minor version
+    required: False
+    default: False
+    type: bool
+
+author:
+    - Michele Costa  (@nocturnalstro)
+"""
+
+EXAMPLES = r"""
+- name: Query OpenShift architectures
+  openshift_versions:
+    openshift_version: 4.16.19
```

Comment:
Should not this be just `version`  instead of `openshift_versions`, please refer https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openshift/assisted-service/master/swagger.yaml#/versions/v2ListSupportedOpenshiftVersions